pykafka/simpleconsumer.py

```diff
@@ -849,7 +868,7 @@ def build_offset_commit_request(self):
             self.partition.id,
             self.last_offset_consumed + 1,
             int(time.time() * 1000),
-            b'pykafka'
+            get_bytes('{}'.format(json.dumps(self._offset_metadata)))
```

Comment:
Ya know, if this function gets called a lot, it would probably be better to store the JSON string version of `self._offset_metadata` instead of dumping again every time, since that's super slow.